src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.cpp

```diff
@@ -1958,8 +1959,194 @@ void jit_soft_sign_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, con
 }
 
 void jit_soft_sign_emitter::register_table_entries() {
-    push_arg_entry_of("one", 0x3f800000, true);
+    push_arg_entry_of("one", CONST_1_F, true);
     push_arg_entry_of("positive_mask", 0x7fffffff, true);
 }
+
+/// IS_FINITE ///
+template <>
+void jit_is_finite_emitter::emit_isa<x64::avx512_core>(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    auto vmm_src = Zmm(in_vec_idxs[0]);
+    auto vmm_dst = Zmm(out_vec_idxs[0]);
+    auto &ones_mask = h->k1;
+    auto reg32_one = Reg32(aux_gpr_idxs[0]);
+
+    h->mov(reg32_one, CONST_1_F);
+    h->vfpclassps(ones_mask, vmm_src, 0B10111001);
+    h->knotd(ones_mask, ones_mask);
+    h->vpbroadcastd(vmm_dst | ones_mask | h->T_z, reg32_one);
```

Comment:
Have you compared the perf with the impl for AVX2/SSE42? Not sure that this one is faster

Reply:
Discussed offline.